source/iostreams.tex

```diff
@@ -4521,12 +4521,12 @@
 namespace std {
   template<class charT, class traits>
   class basic_istream<charT, traits>::sentry {
-    bool ok_;                   // \expos
+    bool @\exposid{ok_}@;                   // \expos
```

Comment:
Should we drop the underscore on this name? There's no `sentry::ok()` member function it could clash with, and if it's in kebab font it's clearly not a public name.

Reply:
Personally, I would prefer if we would differentiate things beyond the font choice. (The font differentiation is a good additional helpful clue, so keep that.) I understand that already fails for some grammar terms and source code names, but given a choice (such as for exposition-only data members), having an underscore suffix for data members feels like a good general differentiation.

@tkoeppe , what do you think?

Reply:
We kept the underscores in #6433